doc/tesseract.1.asc

```diff
@@ -90,15 +90,16 @@ OPTIONS
 	contains a list of variables and their values, one per line, with a
 	space separating variable from value.  Interesting config files
 	include: +
-  * `hocr` - Output in hOCR format (file extension `.hocr`).
-  * `pdf` - Output PDF (file extension `.pdf`).
-  * `tsv` - Output TSV (file extension `.tsv`).
-  * `txt` - Output plain text (file extension `.txt`).
+  * `alto` - Output in ALTO format (file extension `.xml`).	
   * `get.images` - Write images.
+  * `hocr` - Output in hOCR format (file extension `.hocr`).
```

Comment:
I'd keep the "Output" config files together, not mix them with config files which are rarely used.